docs/dg/dev/backend-development/opentelemetry/overview.md

```diff
@@ -0,0 +1,345 @@
+---
```

Comment:
We need to add data about New Relic + OTEL

Reply:
What do you mean?

Reply:
`It's impossible to have NewRelic and OTel enabled on one environment.`